scripts/update_version.py

```
@@ -0,0 +1,582 @@
+        
+        # Find the position after the header
+        header_end = content.find('\n## [')
+        if header_end == -1:
+            header_end = content.find('\n##')
+        
+        if header_end == -1:
+            # No existing entries, add after title
+            header_end = content.find('\n', content.find('# Changelog'))
+        
+        # Create new entry with proper formatting
+        date = datetime.now().strftime('%Y-%m-%d')
```

Ruff (0.11.9)

`datetime.datetime.now()` called without a `tz` argument

(DTZ005)